adminSiteClient/imagesHelpers.ts

```diff
@@ -1,10 +1,18 @@
+import * as Figma from "figma-api"
 import { RcFile } from "antd/es/upload/interface.js"
 import { Admin } from "./Admin"
 import { DbEnrichedImageWithUserId } from "@ourworldindata/types"
-import { CLOUDFLARE_IMAGES_URL } from "../settings/clientSettings"
+import {
+    CLOUDFLARE_IMAGES_URL,
+    FIGMA_API_KEY,
+} from "../settings/clientSettings"
```

Comment:
Aha so actually, all `clientSettings` are bundled in `owid.mjs` which means that yeah, we _should_ proxy this via the admin server - the potential risk for this key leaking seems low risk as it's read-only, but if you can refactor this in less than a day I think it'd be worth it 👍

Reply:
oh, good to know! glad we caught this now :) 9 out of 10 ten times my anxieties are in the way, but sometimes...